x-pack/plugins/observability_solution/apm/public/components/routing/templates/apm_main_template/entities_inventory_callout.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiLink, EuiToolTip, EuiButtonIcon } from '@elastic/eui';
+import { TechnicalPreviewBadge } from '@kbn/observability-shared-plugin/public';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+import { ApmPluginStartDeps } from '../../../../plugin';
+
+interface EntitiesInventoryCalloutProps {
+  onDissmiss: () => void;
+}
+
+export function EntitiesInventoryCallout({ onDissmiss }: EntitiesInventoryCalloutProps) {
+  const { services } = useKibana<ApmPluginStartDeps>();
+  const { observabilityShared } = services;
+
+  const entitiesInventoryLocator = observabilityShared.locators.entitiesInventory;
```

Comment:
```suggestion
  const entitiesInventoryLocator = observabilityShared.locators.entitiesInventory;
  if(entitiesInventoryLocator === undefined){
    return null;
  }
```

Reply:
wait I thought I saw a comment for this already?

Reply:
You are right! 

We can leave it as:

```
href={entitiesInventoryLocator.useUrl({})}